src/logger/loggerConfig.c

```diff
@@ -282,34 +288,19 @@ int encodeSampleRate(int sampleRate)
         return SAMPLE_5Hz;
     case 1:
         return SAMPLE_1Hz;
-    default:
     case 0:
+    default:
         return SAMPLE_DISABLED;
     }
 }
 
-// STIEG: FIX ME!  This can be done with math.
-int decodeSampleRate(int sampleRateCode)
+int decodeSampleRate(int rate_code)
 {
-
-    switch(sampleRateCode) {
-    case SAMPLE_200Hz:
-        return 200;
-    case SAMPLE_100Hz:
-        return 100;
-    case SAMPLE_50Hz:
-        return 50;
-    case SAMPLE_25Hz:
-        return 25;
-    case SAMPLE_10Hz:
-        return 10;
-    case SAMPLE_5Hz:
-        return 5;
-    case SAMPLE_1Hz:
-        return 1;
-    default:
-    case SAMPLE_DISABLED:
-        return 0;
+    if (rate_code == 0) {
+        return SAMPLE_DISABLED;
+    }
+    else {
+        return TICK_RATE_HZ / rate_code;
     }
```

Comment:
I love ternary operators! but the one time I opt for it to be more readable ;)  

I'll implement your suggestion.